.eslintrc.js

```diff
@@ -34,5 +64,80 @@ module.exports = {
      * https://eslint.org/docs/rules/func-names
      */
     'linebreak-style': 'off',
+
+    /**
+     * The following are eslint rules from the promise-plugin
+     * https://github.com/xjamundx/eslint-plugin-promise
```

Comment:
Question: why not use the recommended set vs. manually enabling all these?

```
{
  "extends": ["plugin:promise/recommended"]
}
```

Are you using ones other than the recommended?

Reply:
What if we do a bit of both here: Extend the recommended set, and for all those ones that aren't in recommended that you want from the [rules set](https://github.com/xjamundx/eslint-plugin-promise#rules), add them manually to the rules below.

Reply:
This looks undone to me, are you going to modify this further?